src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -515,6 +520,11 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
       collections: [catalogCollectionId],
       limit: maxCount,
     };
+
+    if (maxCount === null) {
+      delete payload.limit;
+    }
```

Comment:
I would flip this `if`, so it's more consistent with others below.

Reply:
Was changed as suggested